docs/changelog/3_x.rst

```diff
@@ -0,0 +1,726 @@
+           $(grep "VERSION_CODENAME=" /etc/os-release | cut -d= -f2) testing \
+           | sudo tee /etc/apt/sources.list.d/edgedb.list
+
+   Use this command for installation under Debian/Ubuntu:
+
+   .. code-block:: bash
+
+       $ sudo apt-get update && sudo apt-get install edgedb-3-beta1
+
+   Under CentOS/RHEL, use this installation command:
+
+   .. code-block:: bash
+
+       $ sudo yum install edgedb-3-beta1
+
+   In any required ``systemctl`` commands, replace ``edgedb-server-2`` with
+   ``edgedb-server-3``.
+
+   Under any Docker setups, supply the ``3.0-beta.1`` tag.
+
+2. Take your application offline, then dump your v2.x database with the CLI
+
+   .. code-block:: bash
+
+       $ edgedb dump --dsn <old dsn> --all --format dir my_database.dump/
+
+   This will dump the schema and contents of your current database to a
+   directory on your local disk called ``my_database.dump``. The directory name
+   isn't important.
+
+3. Restore the empty v3.x instance from the dump
+
+   .. code-block:: bash
+
+       $ edgedb restore --all my_database.dump/ --dsn <new dsn>
+
+   Once the restore is complete, update your application to connect to the new
+   instance.
+
+   This process will involve some downtime, specifically during steps 2 and 3.
+
+**Pre-1.0 Instances**
+
+If you're still running pre-1.0 EdgeDB instances (e.g., 1.0-beta3) and want
+to upgrade to 3.0, you would need to upgrade to version 2.x first, followed by
+another upgrade to 3.0, both using the same dump-and-restore process.
+
+
+Client libraries
+----------------
+
+Many of the client libraries have gained code generation capabilities since our
+2.0 release. Look for new releases of all of our client libraries soon which
+will support all 3.0 features.
+
+
+New features
+============
+
+Simplified SDL syntax
+---------------------
+
+As part of our commitment to delivering the best developer experience in
+databases, we've made our schema definition language (or SDL) easier to use.
+You're no longer required to use the ``property`` or ``link`` keywords for
+non-computed properties and links. Also, we've replaced arrows with colons for
+a cleaner look that's easier to type.
+
+This change paves the way for a future syntax for declaring ad-hoc types in
+queries and functions. (Read more about it in `the free types RFC
+<https://github.com/edgedb/rfcs/blob/master/text/1022-freetypes.rst>`_.)
+
+That means that this type definition:
+
+.. code-block:: sdl
+
+    type User {
+      required property email -> str;
+      multi link friends -> User;
+    }
+
+could be replaced with this equivalent one in EdgeDB 3+:
+
+.. code-block:: sdl
+
+    type User {
+      required email: str;
+      multi friends: User;
+    }
+
+Selecting "v3" from the version dropdown in the sidebar will update SDL code in
+versioned sections of the documentation to the new syntax.
+
+
+Query plan
+----------
+
+Among other improvements, the UI now includes a visual query planner to help
```

Comment:
```suggestion
Among other improvements, the UI now includes a visual query analyzer to help
```

Reply:
@elprans Should the title of this section also change from "Query plan" to "Query analysis" or "Query analyzer?"

Reply:
Query performance analysis